source/customizing-application/editor-ui/creating-custom-editor.rst

```diff
@@ -889,3 +889,64 @@ Child elements:
 
 - **template**, Optional element (see :ref:`creating-custom-editor-template`)
 
+
+.. _common_advanced_editor_fields:
```

Comment:
Metadata editor doc is based on https://github.com/geonetwork/core-geonetwork/blob/master/schemas/config-editor.xsd so you need to add it here. Probably around https://github.com/geonetwork/core-geonetwork/blob/master/schemas/config-editor.xsd#L1744

I'll check if instruction are missing to build the rst file from the XSD.

Reply:
Do I understand correctly that 'creating-custom-editor.rst' is auto generated from 'config-editor.xsd'? So you suggest I move these changes to the xsd document?

Reply:
Yes using https://github.com/geonetwork/core-geonetwork/blob/master/docs/schema-doc/pom.xml which was probably only used once. Will try to resurrect it.